admin/metabox/class-metabox.php

```diff
@@ -328,7 +328,7 @@ private function get_content_meta_section() {
 			'<span class="yst-traffic-light-container">' . $this->traffic_light_svg() . '</span>',
 			$tabs,
 			array(
-				'link_title' => __( 'Content', 'wordpress-seo' ),
+				'link_title' => __( 'Content:', 'wordpress-seo' ),
```

Comment:
Not sure if we want the semicolon translatable. @tacoverdo could you give some feedback on this?
Otherwise the change would be very simple though: `__( 'Content', 'wordpress-seo' ) . ':'`